scripts/isolinux.cfg

```diff
@@ -1,5 +1,9 @@
-default rancheros
-label rancheros
-    kernel /boot/vmlinuz
-    initrd /boot/initrd
-    append quiet rancher.autologin=tty1 rancher.autologin=ttyS0
+TIMEOUT 20  #2s
+#PROMPT 1
+
+# doesn't appear to work here?
+INCLUDE ../global.cfg
+
+# each INCLUDEd file has a `DEFAULT mylabel` in it, and the last one wins
+INCLUDE ../linux-previous.cfg
```

Comment:
what I want to do is use the same kernel&initrd cfg file for both iso and installed boot - and I am thinking of using previous on the iso as the recovery option